Yoast/wordpress-seo
# Wincher upgrade callouts

## Context

* Add the Wincher account upgrade callouts in both edit-post and dashboard pages.

## Summary


This PR can be summarized in the following changelog entry:

* Add the Wincher upgrade callout to the Wincher performance report on the dashboard.
* Add the Wincher upgrade callout to the Wincher SEO performance on the edit post page.

## Test instructions
- Have a wincher account, add 4 keywords to Wincher.
- Go to `Yoast SEO`->`Integration` and have Wincher integration enabled.
- Go to the edit post page.
- Add a keyphrase in "Focus keyphrase" and then enable tracking it in the "Track SEO performance" in the meta box.
- Check an upgrade callout is displayed above the keyphrases table:
![image](https://user-images.githubusercontent.com/129404933/235740598-5e5044bd-7d02-4b5a-878d-c8fd021e66ca.png)
- If it's not a free wincher account, we don't show "free" in the title on the edit-post page 
- Go to the dashboard page.
- Check an upgrade callout is displayed above the keyphrases table, with a shortened title this time.
![image](https://user-images.githubusercontent.com/129404933/235741603-42221f57-23f4-4806-a271-5c7ffa3d50cc.png)

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes https://github.com/Yoast/wordpress-seo/pull/20239
